src/sentry/taskworker/registry.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import logging
+import time
+from collections.abc import Mapping
+from datetime import timedelta
+from typing import Any
+from uuid import uuid4
+
+import orjson
+from arroyo.backends.kafka import KafkaPayload, KafkaProducer
+from arroyo.types import Topic as ArroyoTopic
+from google.protobuf.timestamp_pb2 import Timestamp
+from sentry_protos.sentry.v1alpha.taskworker_pb2 import RetryState, TaskActivation
```

Comment:
```suggestion
from sentry_protos.sentry.v1.taskworker_pb2 import RetryState, TaskActivation
```

Once the new protos package is released.